.github/workflows/Packages - EdFi.Ods.Minimal.Template.PostgreSQL.yml

```diff
@@ -13,7 +13,7 @@ on:
   repository_dispatch:
     types: [triggered-from-implementation-repo]
 env:
-  EDFI_STANDARD_REFERENCE: "4.0.0-a"
```

Comment:
This comment applies to this workflow and all the other workflows with this change:

- Could we instead keep this env var and just change the value to `main`? This way the next time when we need to point this to a tag, we just update it to the appropriate value instead of having it add it back in two places
- Additionally, do we need the Is pull request step? I thought in TeamCity we usually kept the DS repo pointed to a tag or main.

Reply:
Fixed it .   There is PR in DS repo https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Standard/pull/142  which excluding fianance.xml since we have bug in that file . Don't Merge it .  https://tracker.ed-fi.org/browse/ODS-5629 needs to be merged first and then next set of DataStandard ODS ticket , we will generate template builds along with finance sample data in ODS